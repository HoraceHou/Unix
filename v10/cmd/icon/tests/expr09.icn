record array(a,b,c,d,e,f,g)

procedure dummy(u,v,x,y,z)
   suspend u | v
   return x
end

procedure main()
   write("?0 ----> ",image(?0) | "none")
   write("?10 ----> ",image(?10) | "none")
   write("?10 ----> ",image(?10) | "none")
   write("?10 ----> ",image(?10) | "none")
   write("?20 ----> ",image(?20) | "none")
   write("?[1,2,3,4] ----> ",image(?[1,2,3,4]) | "none")
   write("?[1,2,3,4] ----> ",image(?[1,2,3,4]) | "none")
   write("x := array(1,2,3,4,5,6,7) ----> ",image(x := array(1,2,3,4,5,6,7)) | "none")
   write("?x ----> ",image(?x) | "none")
   write("?x ----> ",image(?x) | "none")
   write("?x ----> ",image(?x) | "none")
   write("?x ----> ",image(?x) | "none")
   write("?x ----> ",image(?x) | "none")
   write("?x ----> ",image(?x) | "none")
   write("?x ----> ",image(?x) | "none")
   write("?x ----> ",image(?x) | "none")
   write("?&null ----> ",image(?&null) | "none")
end
