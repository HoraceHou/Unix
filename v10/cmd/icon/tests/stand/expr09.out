?0 ----> 0.21132487
?10 ----> 5
?10 ----> 4
?10 ----> 6
?20 ----> 9
?[1,2,3,4] ----> 2
?[1,2,3,4] ----> 1
x := array(1,2,3,4,5,6,7) ----> record array(7)
?x ----> 6
?x ----> 1
?x ----> 6
?x ----> 1
?x ----> 3
?x ----> 4
?x ----> 5
?x ----> 5
Run-time error 113 at line 25 in expr09.icn
invalid type to random operation
offending value: &null
